src/Video/Renderer.hpp

```diff
@@ -35,6 +35,16 @@ namespace Video {
             
             /// Clear the previous frame's data.
             void Clear();
+
+            /// Render a static mesh.
+            /**
+            * @param geometry The geometry to render.
+            * @param viewMatrix The camera's view matrix.
+            * @param projectionMatrix The camera's projection matrix.
+            * @param modelMatrix Model matrix.
+            */
+
```

Comment:
We ought to remove this empty line so it's clear that the doxygen comment belongs to the function.